code/modules/detectivework/tools/samplekit.dm

```diff
@@ -0,0 +1,72 @@
+/obj/item/forensics
+	icon = 'icons/obj/forensics.dmi'
+	flags = NOBLUDGEON
+
+/obj/item/forensics/proc/handlefiber(ftext)
+	return replacetext(replacetext(replacetext(ftext, "material from a pair of ", ""), ".", ""), "Fibers from a", "")
```

Comment:
Try:
`return regex("REGEX").Replace(ftext, "")`
Replacing REGEX with the correct regexp and escaping the \.